windows/asio.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+        asio.cpp
+
+        asio functions entries which translate the
+        asio interface to the asiodrvr class methods
```

Comment:
As mentioned by @hoffie 
How can we avoid including code based on Steinberg's work? Could you at least show clearly what you changed (and why)

Reply:
@ann0see 

> As mentioned by @hoffie How can we avoid including code based on Steinberg's work? Could you at least show clearly what you changed (and why). We should still require the Steinberg code to be downloaded (this ensures we're compatible with Debian for example)

I already changed the code to include the ASIOSDK2 as before, but just using two headers from the common folder. Also I rearanged some of my code and changed the filenames so they don't match SDK filenames to avoid confusion.